assets/sass/licences-pdf.scss

```diff
@@ -183,7 +183,40 @@ img {
   padding-bottom: 50px;
 }
 
+.extraLargePaddingBottom {
```

Comment:
Is there any reason this needs to be camel case rather than kebab case as the others are?

Reply:
no particular reason .
i'll change it to hyphenated